client/src/about_page_components/AboutMetadata.js

```diff
@@ -0,0 +1,119 @@
+import React from "react";
+import Card from "@mui/material/Card";
+import CardActions from "@mui/material/CardActions";
+import CardContent from "@mui/material/CardContent";
+import Typography from "@mui/material/Typography";
+
+import {
+  CardHeader,
+  Collapse,
+  Divider,
+  IconButton,
+  styled,
+} from "@mui/material";
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+import CircleIcon from "@mui/icons-material/Circle";
+
+const ExpandMore = styled((props) => {
+  const { expand, ...other } = props;
+  return <IconButton {...other} />;
+})(({ theme, expand }) => ({
+  transform: !expand ? "rotate(0deg)" : "rotate(180deg)",
+  marginLeft: "auto",
+  transition: theme.transitions.create("transform", {
+    duration: theme.transitions.duration.shortest,
+  }),
+}));
+
+function AboutMetadata() {
+  const [expanded, setExpanded] = React.useState(true);
+  const [expanded2, setExpanded2] = React.useState(false);
+
+  let node = { id: "id3", name: "Lily", val: 3, color: "#b2df8a" };
+
+  const handleExpandClick = () => {
+    setExpanded(expanded);
```

Comment:
If I understand correctly, I think this should be ```setExpanded(!expanded)```

Reply:
The card width changes when it expands and explicitly setting the width made it look weird so I made sure one is always open. That way the card is always the same width. . I can remove that function since it's redundant.